assets/css/app.css

```diff
@@ -106,6 +106,10 @@
   animation: 0.2s ease-out 0s normal forwards 1 fade-out-keys;
 }
 
+.flex-col-reverse {
```

Comment:
idk why I had to add this, because it's documented in tailwind, but it didn't seem to apply. Could be a version thing?